server/src/main/java/org/apache/druid/indexing/overlord/SegmentPublishResult.java

```diff
@@ -34,59 +34,59 @@
 import java.util.Set;
 
 /**
- * Result of an operation that attempts to publish segments. Indicates the set of segments actually published
- * and whether or not the transaction was a success.
- *
- * If "success" is false then the segments set will be empty.
- *
- * It's possible for the segments set to be empty even if "success" is true, since the segments set only
- * includes segments actually published as part of the transaction. The requested segments could have been
- * published by a different transaction (e.g. in the case of replica sets) and this one would still succeed.
+ * Result of a segment publish operation.
  */
 public class SegmentPublishResult
 {
   private final Set<DataSegment> segments;
   private final boolean success;
-  @Nullable
+  private final boolean retryable;
   private final String errorMsg;
```

Comment:
The errorMsg was Nullable before and is not Nullable now. Is this expected ?

Reply:
Yeah, this particular annotation doesn't do much. Just causes unnecessary warnings in the IDE.

The constructor arg and the getter are still marked as `@Nullable`.